Files/Helpers/Convert/StorageItemHelpers.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Files.Filesystem;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+using Windows.Storage;
+
+namespace Files.Helpers
+{
+    public static class StorageItemHelpers
+    {
+        public static async Task<IStorageItem> ToStorageItem(this string path)
+        {
+            StorageFolder folder = await StorageFolder.GetFolderFromPathAsync(Path.GetDirectoryName(path));
```

Comment:
This is likely to throw exceptions, should probably be wrapped with `FilesystemTasks.Wrap()` and explicit error handling should be added whenever it's used